app/src/main/res/layout/revision_card_activity.xml

```diff
@@ -35,6 +35,16 @@
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
           android:layout_marginEnd="20dp"/>
+
+        <ImageButton
+          android:id="@+id/action_bottom_sheet_options_menu"
+          style="?android:actionOverflowButtonStyle"
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
```

Comment:
Does it need contentDescription for screen-readers?

Reply:
If yes, please add test too.

Reply:
As mentioned earlier I don't think we need to add contentDescription explicitly.